packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -186,6 +186,17 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the component should render in compact mode. Set to false by default */
   compact?: boolean;
+
+  /** 
+   * boolean to control if pages containing unchanged items should be cached, this is a perf optimization 
+   * The same property in List.Props
+   */
+  usePageCache?: boolean;
+  
+  /** 
+   * when items count is not greater than this number, List will render in fast mode without expensive measurement operations that cause reflows
```

Comment:
Consider replacing with:

```tsx
onShouldVirtualize: (props: IDetailsListProps) => boolean;
```

Also documentation needs to be clear:

"Optional callback to determine whether the list should be rendered in full, or virtualized. Virtualization will add and remove pages of items as the user scrolls them into the visible range. This benefits larger list scenarios by reducing the DOM on the screen, but can negatively affect performance for smaller lists. The default implementation will virtualize when rendering more than 50 items."